avm/utl/types/avm-common-types/README.md

```diff
@@ -0,0 +1,291 @@
+# Default interface types for AVM modules `[Types/AvmCommonTypes]`
+
+This module provides you with all common variants for AVM interfaces to be used in AVM modules.
+
+Details for how to implement these interfaces can be found in the AVM documentation [here](https://azure.github.io/Azure-Verified-Modules/specs/bicep/interfaces).
```

Comment:
The actual main link for interfaces is https://azure.github.io/Azure-Verified-Modules/specs/shared/interfaces/

Reply:
Good find. Fixed